python/sglang/test/test_choices.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
```

Comment:
This test looks good!
Can you move it under https://github.com/sgl-project/sglang/tree/main/test/lang and write with the python package unittest, similar to other files in that folder. We will run this in CI.